src/screens/Saved.js

```diff
@@ -1,33 +1,35 @@
-import React, { Component, Fragment } from 'react';
+import React, { Component, Fragment } from "react";
```

Comment:
In general for RN the use of ' vs " is just a stylistic choice, whatever you choose just make sure it's consistent